packages/yarnpkg-core/sources/Project.ts

```diff
@@ -44,7 +44,7 @@ import {IdentHash, DescriptorHash, LocatorHash, PackageExtensionStatus} from './
 // When upgraded, the lockfile entries have to be resolved again (but the specific
 // versions are still pinned, no worry). Bump it when you change the fields within
 // the Package type; no more no less.
-const LOCKFILE_VERSION = 7;
+const LOCKFILE_VERSION = 8;
```

Comment:
Did you mean to bump the cache key instead?

Reply:
No; I use the lockfile version to know whether the project was last installed by an older Yarn version and migrate its settings. We are at version 6 in stable, 7 in rc, so bumping to 8 also helps the few who already migrated to 4.x.

Reply:
Ah, that makes sense.